### PR TITLE
delete selected clip

### PR DESCRIPTION
**Pull request by krizmyass**
added option to delete selected clip by long pressing right button on page_playback

### CODE REVIEW

**Comment by bkleiner** [comment]
i am fine with going this solution in the interims, but ultimately i think we want a solution like https://github.com/hd-zero/hdzero-goggle/issues/98 to access more advanced options in a given menu. having 5 different keystrokes to trigger different options is gonna get confusing quick.

**Comment by krizmyass** [comment]
> i am fine with this in the interims, but ultimately i think we want a solution like #98 to access more advanced options in a given menu. having 5 different keystrokes to trigger different options is gonna get confusing quick.

Great. I will work on a context menu.

File: src/ui/page_playback.c [hunk 1]
```diff
@@ -308,6 +308,32 @@ static void mark_video_file(int seq) {
     update_page();
 }
 
+static void delete_video_file(int seq) {
+    media_file_node_t *pnode = get_list(seq);
+            if (!pnode) {
```
Comment (bkleiner) [inline src/ui/page_playback.c:313]: whitespace, also in following, please use vscode's clangbased autoformat

File: src/ui/page_playback.c [hunk 2]
```diff
@@ -308,6 +308,32 @@ static void mark_video_file(int seq) {
...
+                perror("delete_video_file failed. (PNODE ERROR)");
```
Comment (bkleiner) [inline src/ui/page_playback.c:314]: please use `LOGE`

File: src/ui/page_playback.c [hunk 3]
```diff
@@ -308,6 +308,32 @@ static void mark_video_file(int seq) {
...
+    char cmd[128];
+    sprintf(cmd, "rm %s/%s.*", MEDIA_FILES_DIR, pnode->label);
+
+    if (system(cmd) != -1) {
+
+        walk_sdcard();
+        media_db.cur_sel = constrain(seq, 0, (media_db.count - 1));
+        update_page();
+
+        perror("delete_video_file successful.");
```
Comment (bkleiner) [inline src/ui/page_playback.c:327]: not an error please use `LOGD`

File: src/ui/page_playback.c [hunk 4]
```diff
@@ -308,6 +308,32 @@ static void mark_video_file(int seq) {
...
+
+    } else {
+
+        perror("delete_video_file failed.");
```
Comment (bkleiner) [inline src/ui/page_playback.c:331]: please use `LOGE`

**Review by bkleiner — CHANGES_REQUESTED** [review]

**Review by bkleiner — APPROVED** [review]

**Comment by bkleiner** [comment]
finally got around to testing this - seems to work fine.

**Pull request merged**